packages/devtools_app/lib/src/memory/flutter/memory_protocol.dart

```diff
@@ -259,6 +303,80 @@ class ClassHeapDetailStats {
       'count: $instancesCurrent, bytes: $bytesCurrent]';
 }
 
+// TODO(terry): Need the iOS version of this data.
+/// Android ADB dumpsys meminfo data.
+class AdbMemoryInfo {
+  AdbMemoryInfo(this.realtime, this.javaHeap, this.nativeHeap, this.code,
+      this.stack, this.graphics, this.other, this.system, this.total);
+
+  factory AdbMemoryInfo.fromJson(Map<String, dynamic> json) => AdbMemoryInfo(
+        json['Realtime'] as int,
+        json['Java Heap'] as int,
+        json['Native Heap'] as int,
+        json['Code'] as int,
+        json['Stack'] as int,
+        json['Graphics'] as int,
+        json['Private Other'] as int,
+        json['System'] as int,
+        json['Total'] as int,
+      );
+
+  Map<String, dynamic> toJson() => <String, dynamic>{
+        'Realtime': realtime,
+        'Java Heap': javaHeap,
+        'Native Heap': nativeHeap,
+        'Code': code,
+        'Stack': stack,
+        'Graphics': graphics,
+        'Private Other': other,
+        'System': system,
+        'Total': total,
+      };
+
+  /// Create an empty AdbMemoryInfo (all values are)
+  static AdbMemoryInfo empty() => AdbMemoryInfo(0, 0, 0, 0, 0, 0, 0, 0, 0);
+
+  /// Number of milliseconds since the device was booted (value zero) including deep
+  /// sleep.  This clock is guaranteed to be monotonic, and continues to tick even
+  /// in power saving mode. The value zero is Unix Epoch UTC (Jan 1, 1970 00:00:00).
+  /// This DateTime, from USA PST, would be Dec 31, 1960 16:00:00 (UTC - 8 hours).
+  final int realtime;
+
+  final int javaHeap;
+
+  final int nativeHeap;
+
+  final int code;
+
+  final int stack;
+
+  final int graphics;
+
+  final int other;
+
+  final int system;
+
+  final int total;
+
+  DateTime get realtimeDT => DateTime.fromMillisecondsSinceEpoch(realtime);
+
+  /// Duration the device has been up since boot time.
+  Duration get bootDuration => Duration(milliseconds: realtime);
+
+  @override
+  String toString() => '  realtime=$realtime\n'
```

Comment:
nit: use `:` and remove the leading  `' . '` from each line in the toString.
The toString should probably also list what this className is for easier debugging.
Also feel free to just remove the toString if you don't need it

Reply:
Done.